script/gitlabci/job_base.yml

```diff
@@ -108,4 +103,3 @@
   tags:
     - amd
     - rocm
-
```

Comment:
Unnecessary whitespace change?

Reply:
I removed a useless empty line, not required for the correctness but two empty lines at EOF look bad^^

Reply:
ok fine!